packages/eslint-plugin/docs/rules/no-floating-promises.mdx

```diff
@@ -82,6 +83,24 @@ await Promise.all([1, 2, 3].map(async x => x + 1));
 </TabItem>
 </Tabs>
 
+## Handling Promises rejections
```

Comment:
Oh, sorry. I feel I may have led you in the wrong direction here. I'm thinking, for now, for this PR, let's not add this section (sorry for the time you spent on this!). 

But it _would_ be in scope to prominently link to the MDN error-handling writeup, maybe by a simple `See [the MDN docs](link) for a detailed writeup on promise error-handling.` right below the list of ways to satisfy the rule?

Reply:
I love writing things and explanations for people to read.

I studied this paragraph with the idea of it linking to MDN, not replacing it, but also to share the knowledge we discussed in here and adding some values related to this rule.

I think that removing such paragraph and reducing it to be just a link, would almost completely void (_pun intended_) the purpose of this PR, which I guess could be closed then.

We would be going to remove the whole concept of "looks like to be handled" and "guardrails" you highlighted before in this discussion, which are two concepts that I think are worth to me mentioned in terms of mental model.

I can move it below the list, for sure, and I can tune the paragraph a bit.

However, I'm of the idea I already spent too much energy on this hoping it would be okay and reducing it to a link makes me feel I wasted my time and energy. I hope you understand how I'm feeling after this comment.

Reply:
I tried to rework the paragraph a little bit and moved it up in the discussion. Let me know how it feels now.

Reply:
@alexandercerutti, I hear you. But, it's deliberate that the issue was scoped narrowly to be essentially about the `void` wording only, see https://github.com/typescript-eslint/typescript-eslint/issues/9947#issuecomment-2334294112. We have #8088 and #8517 already to address the larger scale efforts of improving the promise docs, deep-diving on the technical details, and ensuring that the various promise rules' docs are coherent with each other, rather than just this page alone.

So, we all agree that the deep-dive info on promise rejections needs to exist, but, unfortunately, I need to insist that this particular PR is not the time and place to make it happen, sorry.

We'll certainly keep this discussion in mind, though for when it does happen in the linked issues!

Best, Kirk